docs/user/filesystem.md

```diff
@@ -894,3 +894,126 @@ $
 
 While this mode is not expected to be used directly by users, it is useful for
 the unified namespace integration.
+
+## Interception Library `libpil4dfs`
+
+A new interception library called `libpil4dfs` is introduced. It is similar to libioil,
+but it intercepts more APIs, including read/write as well as metadata related functions.
+This provides kernel-bypass for I/O data, leading to improved performance. This is a preview
+version. Some features are not implemented yet.
```

Comment:
please move the last sentence to the limitation section which can be removed / updated once this is not a preview anymore.

Reply:
This sentence is moved to limitation section as suggested. Thank you!